apps/meteor/server/startup/migrations/v299.ts

```diff
@@ -0,0 +1,27 @@
+import { Settings } from '@rocket.chat/models';
+
+import { settingsRegistry } from '../../../app/settings/server';
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 299,
+	async up() {
+		const customOauthServices = await Settings.find({ _id: /Accounts_OAuth_Custom-[^-]+$/im }).toArray();
+		const serviceNames = customOauthServices.map(({ _id }) => _id.replace('Accounts_OAuth_Custom-', ''));
+
+		serviceNames.forEach(async (serviceName) => {
```

Comment:
`Array.forEach` is not `async`.. you can either use `await Promise.all( ... )` with `serviceNames.map` or use `for await`:
```suggestion
		for await (const serviceName of serviceNames) {
```